src/util/logger.js

```diff
@@ -0,0 +1,22 @@
+import pino from 'pino';
+
+
+const isDev = process.env.NODE_ENV !== 'production';
+const isTest = process.env.NODE_ENV === 'test';
+
+const reqSerializerDev = req => `${req.method.toUpperCase()} ${req.url.href}`;
+const resSerializerDev = res => res.statusCode;
+
+const logger = pino({
+	prettyPrint: isDev,
+	timestamp: isDev,
+	messageKey: 'message',
+	enabled: !isTest,
```

Comment:
yeah I'll have another think about this - the main problem is that while `jest --silent` will silence `console.x` output, it _doesn't) silence this logger because this logger uses `process.stdout` directly. I'll see if there's an easy way to conditionally enable it during tests.